package.json

```diff
@@ -70,7 +70,7 @@
     "file-loader": "^6.2.0",
     "filesize": "^10.1.2",
     "fs-extra": "^11.2.0",
-    "glob": "^10.4.2",
+    "glob": "^11.0.0",
```

Comment:
Do we need to update this urgently? It will change the mininum node version to v20.

Reply:
Since webOS build supports node 20.12.2, it seemed to be acceptable.

Reply:
Please update the minimum node version accordingly.